### ISSUE TITLE
Resumed experiments don't have correct authentication to access metrics and heartbeat

### ISSUE THREAD

**Issue by lgeiger** [polyaxon/polyaxon#544]
### Describe the bug

When resuming an experiment that was stopped or crashed for some reason, the resumed experiment doesn't have access to the metrics and heartbeat API endpoints. When trying to write metrics using the Keras callback, it will keep logging the following errors:
```
E0802 http_transport.py:248] Request to http://polyaxon-polyaxon-api:80/api/v1/username/project/experiments/0001/metrics/ failed with status code 401.
Reason: {"detail":"Authentication credentials were not provided."}
E0802 http_transport.py:248] Request to http://polyaxon-polyaxon-api:80/api/v1/username/project/experiments/0001/_heartbeat/ failed with status code 401.
Reason: {"detail":"Authentication credentials were not provided."}
```

### To Reproduce
Resume an experiment and try to use the Keras callback to log metrics.

### Expected behavior
The resumed experiment should have the correct authentication and no error should be logged.

### Environment
Polyaxon 0.5.4
GKE

**Comment by mouradmourafiq** [comment]
Was this ever working before? I have the impression that auth on resumed xps did not work since v0.4.4?

**Comment by lgeiger** [comment]
> Was this ever working before? I have the impression that auth on resumed xps did not work since v0.4.4?

I think we haven't seen any issues in 0.4.4, but I am not sure since we only recently started interacting with the tracking API.

**Comment by mouradmourafiq** [comment]
I will cherry-pick the commit and add it as a patch to the v0.5.5 release, I think the auth issue here goes back to 0.3.9.

**Comment by lgeiger** [comment]
Thanks for the fast fix

**Comment by waltsims** [comment]
> > Was this ever working before? I have the impression that auth on resumed xps did not work since v0.4.4?
> 
> I think we haven't seen any issues in 0.4.4, but I am not sure since we only recently started interacting with the tracking API.

FYI: we are still experiencing this in v0.4.4. We will upgrade to v0.5.5.

**Comment by waltsims** [comment]
> > Was this ever working before? I have the impression that auth on resumed xps did not work since v0.4.4?
> 
> I think we haven't seen any issues in 0.4.4, but I am not sure since we only recently started interacting with the tracking API.

FYI: we are still experiencing this in v0.4.4 when resuming an experiment that was previously a member of a group. Metrics are therefore no longer printed. We will upgrade to v0.5.5 at the beginning of next week.

**Comment by mouradmourafiq** [comment]
A regression has been introduced while fixing the issue, see #553. It's quite straightforward to fix, and it will be in the v0.5.6 version.

Good news is that, the spawning part of Polyaxon is going through a lot of optimizations, and will have an extensive amount of testing in the v0.6.

**Comment by waltsims** [comment]
I saw #533. Glad to hear it. Just wanted to inform all here.
